config.yaml

```diff
@@ -19,6 +19,11 @@ verifier:
   verification_key_path: "./verification_key.json"
   allowed_age: 18
   allowed_citizenships: ["UKR"]
+# during proof verification allowed_credentials_count OR allowed_credential_timestamp will be checked in public signals
+# allowed_credentials_count - amount of credentials that user may create if the credentials timestamp after airdrop starting
+# allowed_credential_timestamp - timestamp that limits when user might create credentials if it has more than 1
+  allowed_credentials_count: 1
```

Comment:
what is relation of this to 'credentials'? it's about identities, revocation, reissuance